src/services/api/DatagouvfrAPI.ts

```diff
@@ -160,17 +181,19 @@ export default class DatagouvfrAPI {
    * Delete an entity (DELETE)
    *
    * @param {string} entityId - A UUID entity id
-   * @returns {Promise}
+   * @returns {Promise<Response>}
    */
-  async delete(entityId) {
-    return this.httpClient.delete(`${this.url()}/${entityId}/`).then(
+  async delete(entityId: string): Promise<Response> {
+    return await this.httpClient.delete(`${this.url()}/${entityId}/`).then(
       (response) => response,
       (error) => this.#handleError(error)
     )
   }
 
-  #handleError({ response, message }) {
-    if (response) return { status: response.status }
-    return { message }
+  #handleError({ response, message }): Response {
```

Comment:
@edelagnier vu que c'est du TypeScript maintenant, ne pourrait-on utiliser `private` plutôt ?

```suggestion
  private handleError({ response, message }): Response {
```